Added datetime picker for date and datetime attribute types

lordblendi opened the pull request:
- Added date picker
- Added datetime picker
- Updated readme

nvdk commented on package.json:
```diff
@@ -48,6 +50,7 @@
     "ember-load-initializers": "^1.0.0",
     "ember-resolver": "^4.0.0",
     "ember-source": "~2.14.1",
+    "jquery-datetimepicker": "^2.5.4",
```
perhaps if we move this to dependencies it may no longer be required to install it separately

lordblendi commented:
Hey @nvdk !

- I added the default value on the new hook -> moment()
- I moved the jquery-datetimepicker to the dependencies, but that wasn't enough. So I simply install it after the default blueprint. See https://github.com/tenforce/ember-mu-application-generator/pull/1/commits/3f406b3197a87a4b0f31dc4375baa1e42289037c#diff-0dbf651fcbc130eac31751f585631e4fR10
- @asjongers had a good idea for the datetime transfrom: just convert it to date.
